python/cmake/FindPETSc4py.cmake

```diff
@@ -45,6 +45,16 @@ if(PETSC4PY_INCLUDE_DIRS)
   set(PETSC4PY_FIND_QUIETLY TRUE)
 endif(PETSC4PY_INCLUDE_DIRS)
 
+message(STATUS ${PYTHON_EXECUTABLE})
```

Comment:
Why is this change here?

Reply:
Ah, I believe I once had some build issues. Overlooked that...